pkg/operator/controllers/pullsecret/pullsecret_controller.go

```diff
@@ -184,15 +184,18 @@ func (r *Reconciler) ensureGlobalPullSecret(ctx context.Context, operatorSecret,
 		// allows for simpler logic flow, when delete and create are not handled separately
 		// this call happens only when there is a need to change, it has no significant impact on performance
 		err := r.client.Delete(ctx, secret)
+		r.log.Info("Pullsecret Not Found, Creating Again")
```

Comment:
It might help to specify what pull secret - maybe something like `Global pull secret not found, will create again`?